content/en/sensitive_data_scanner/_index.md

```diff
@@ -119,14 +168,41 @@ You can create custom scanning rules using regex patterns to scan for sensitive
 {{% sds-scanning-rule %}}
 1. Click **Add Rule**.
 
+[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration
+[2]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_rule
+{{< /collapse-content >}} 
+{{% /tab %}}
+{{% tab "In the Agent" %}}
+
+A scanning rule determines what sensitive information to match within the data defined by a scanning group. You can add **up to 20** predefined scanning rules from Datadog's Scanning Rule Library. The Datadog Agent scans your data in your local environment during log collection, before logs are sent to the Datadog platform.
+
+For Terraform, see the [Datadog Sensitive Data Scanner rule][2] resource.
+
+To add scanning rules, perform the following steps.
+
+<div class="alert alert-warning"><strong>Note</strong>: Sensitive Data Scanner in the Agent supports only predefined scanning rules from Datadog's Scanning Rule Library. The total number of scanning rules is limited to 20.</div>
```

Comment:
Can we move this up, before the "To add scanning rules, perform the following steps:"?